copier/main.py

```diff
@@ -376,13 +377,24 @@ def jinja_env(self) -> SandboxedEnvironment:
         """
         paths = [str(self.template.local_abspath)]
         loader = FileSystemLoader(paths)
+        default_extensions = [
+            "jinja2_ansible_filters.AnsibleCoreFiltersExtension",
+        ]
+        extensions = default_extensions + list(self.template.jinja_extensions)
         # We want to minimize the risk of hidden malware in the templates
         # so we use the SandboxedEnvironment instead of the regular one.
         # Of course we still have the post-copy tasks to worry about, but at least
         # they are more visible to the final user.
-        env = SandboxedEnvironment(loader=loader, **self.template.envops)
-        default_filters = {"to_nice_yaml": to_nice_yaml}
-        env.filters.update(default_filters)
+        try:
+            env = SandboxedEnvironment(
+                loader=loader, extensions=extensions, **self.template.envops
+            )
+        except ModuleNotFoundError as error:
+            raise UserMessageError(
```

Comment:
Please use a specific exception class. See the errors module.

Reply:
Do you mean something like this? 

```python
class ExtensionNotFoundError(UserMessageError, CopierError):
    """Extensions listed in the configuration could not be loaded."""
```

By the way, why do you inherit from both `UserMessageError` *and* `CopierError` when the first already inherits from the second?

Reply:
> Do you mean something like this?

Yes



> By the way, why do you inherit from both `UserMessageError` _and_ `CopierError` when the first already inherits from the second?

Oops, that's a bug.

Reply:
I wrote like that anyway, for the sake of consistency. You'll be able to fix all error classes at once in a later commit/PR :slightly_smiling_face: